catalog/vehicle-single-language-output.schema.json

```diff
@@ -103,8 +99,8 @@
       "label": { "type": ["null","string"], "minLength": 1 }
     }},
     "interiorSimpleColor":  { "type": "object", "additionalProperties": false, "required": ["id", "label"], "properties": {
-      "id": { "$ref": "./base.schema.json#/$defs/simple-color-id" },
-      "label": { "type": "string", "minLength": 1 }
+      "id": { "anyOf": [{"type": "null"}, { "$ref": "./base.schema.json#/$defs/simple-color-id" }] },
```

Comment:
I think, we must use allways snakecase

Reply:
No, this is a limitation of AppSearch. Otherwise, the interface contract uses camelCase.

Reply:
BTW, your comment is on a line of the schema, where on `anyOf` uses camelCase. This is a keyword defined by `json-schema`, so it's not up to us to choose :-)